integration_test/src/binance-native-token.spec.js

```diff
@@ -0,0 +1,36 @@
+const assert = require('assert');
+const worker = require('../../blockchains/eth/eth_worker');
+
+
+describe('BEP20 worker test', function () {
+  it('BEP20 worker should extract blocks from 20000000 to 20000999 including', async function () {
```

Comment:
Second feature in this PR. Add the skeleton for what should become a set of integration tests for each chain. We query a range of blocks from the running Node and compare with how we expect this data to look like.